setup.py

```diff
@@ -40,10 +40,16 @@ def find_scripts(script_dir, prefix):
     packages=find_packages(),
     py_modules='version',
     install_requires=[
+        'astropy',
```

Comment:
It's the same as in the environment file (no version specification).

If astropy is already installed in any version, it won't update, if it is not installed it will install the latest version.

Reply:
I was wondering whether we would need to implement any modification if this was required (as it was done in [ctapipe](https://github.com/cta-observatory/ctapipe/pull/1212)), but I guess that Travis is testing it now with 4.0 since we are not specifying anything, right?

Reply:
Yes, looking into the log, astropy 4 is installed

Reply:
That is a bit strange. Seems it's because of conda-forge.

Reply:
ok, shall we merge this and open another one forcing astropy 4.0 in case there are some things that need to be modified?

Reply:
yes